package.json

```diff
@@ -43,23 +43,22 @@
     "babel-preset-es2015": "^6.9.0",
     "babel-preset-react": "^6.5.0",
     "babel-preset-stage-0": "^6.24.1",
+    "eslint": "^4.4.0",
     "eslint-config-airbnb": "^15.1.0",
     "eslint-plugin-import": "^2.7.0",
     "eslint-plugin-jsx-a11y": "^6.0.2",
     "eslint-plugin-react": "^7.1.0",
-    "eslint": "^4.4.0",
     "pre-commit": "^1.1.2",
-    "react": "^15.6.1",
```

Comment:
Actually even keeping it in the `devDependencies` the example will work only if you first run `npm install` in the project's root